### PR TITLE
[chrony] Move chronyd back to host network namespace

### PR DESCRIPTION
**Pull request by mvasl**
## Description

Enabled `hostNetwork` for chrony pods

## Why do we need it, and what problem does it solve?

This is required for `node-manager`'s NTP collector to be able to access Node's NTP server.

## Why do we need it in the patch release (if we do)?

This PR fixes node time alerts in `node-manager`

## Checklist
- [ ] The code is covered by unit tests.
- [x] e2e tests passed.
- [ ] Documentation updated according to the changes.
- [x] Changes were tested in the Kubernetes cluster manually.

## Changelog entries

```changes
section: chrony
type: fix
summary: Run chrony pods in host network namespace.
impact_level: default
```

### CODE REVIEW

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @mvasl. I've started the workflow for label 'e2e/run/yandex-cloud'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @mvasl. I've started the workflow for label 'e2e/run/openstack'...

File: modules/470-chrony/templates/daemonset.yaml [hunk 1]
```diff
@@ -19,7 +19,7 @@ kind: VerticalPodAutoscaler
 metadata:
   name: chrony
   namespace: d8-{{ .Chart.Name }}
-  {{- include "helm_lib_module_labels" (list . (dict "app" "chrony" "tier" "node" "workload-resource-policy.deckhouse.io" "every-node")) | nindent 2 }}
```
Comment (RomanenkoDenys) [inline modules/470-chrony/templates/daemonset.yaml:22]: Why you remove this ?

File: modules/470-chrony/templates/daemonset.yaml [hunk 2]
```diff
@@ -55,11 +55,11 @@ spec:
         tier: node
         app: chrony
     spec:
-      dnsPolicy: ClusterFirst
+      dnsPolicy: ClusterFirstWithHostNet
+      hostNetwork: true
       imagePullSecrets:
       - name: deckhouse-registry
       {{- include "helm_lib_priority_class" (tuple . "cluster-medium") | nindent 6 }}
-      {{- include "helm_lib_tolerations" (tuple . "any-node") | nindent 6 }}
```
Comment (RomanenkoDenys) [inline modules/470-chrony/templates/daemonset.yaml:62]: why you remove this ?

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @mvasl. I've started the workflow for label 'e2e/run/openstack'...

**Review by RomanenkoDenys — APPROVED** [review]
LGTM

**Comment by deckhouse-BOaTswain** [comment]
Cherry pick PR [6070](https://github.com/deckhouse/deckhouse/pull/6070) to the branch [release-1.52](https://github.com/deckhouse/deckhouse/tree/release-1.52) successful!

**Pull request merged**